ygen/codegen_test.go

```diff
@@ -805,6 +817,9 @@ func TestSimpleStructs(t *testing.T) {
 	}}
 
 	for _, tt := range tests {
+		if tt.name != "enumeration behaviour - multiple enumerations within a union" {
```

Comment:
Stale test skipping code?

BTW -- you can do `go test -run <regexp>` to run an individual test case to avoid this issue when debugging a particular test. If you have spaces in the test name, then they're replaced with `_`.  Subtests can also be referred to by using `TestName/Subtest_Name` as that pattern.

Reply:
Thanks! That's helpful, very useful for debugging enumgen_test.go tests.